mxnet/inference/buildspec.yml

```diff
@@ -31,7 +31,7 @@ images:
  BuildEC2MXNetCPUInferencePy3DockerImage:
    <<: *INFERENCE_REPOSITORY
    build: &MXNET_CPU_INFERENCE_PY3 false
-   image_size_baseline: 2202
+   image_size_baseline: 5000
```

Comment:
have the images doubled in size? why is baseline so high for CPU SM and EC2?

if it's doubled, we should analyze why to make sure we're not including new packages we don't need

Reply:
Thanks for catching this. I am rebuilding images with a smaller baseline size.